src/fairseq2/models/wav2vec2/builder.py

```diff
@@ -139,6 +139,9 @@ class Wav2Vec2EncoderConfig:
     depthwise_conv_kernel_size: int
     """The kernel size of depthwise convolutions in Conformer blocks."""
 
+    depthwise_conv_is_causal: bool
```

Comment:
Same parameter name and docstr (one line below) suggestion as in conformer/convolution.py.